docs/source/_static/custom.css

```diff
@@ -0,0 +1,251 @@
+html {
```

Comment:
Is this ours or is it copied from somewhere?
If copied can we have a link or info where this is coming from?

Reply:
Form Waffle :)